provider/datagen/Cargo.toml

```diff
@@ -104,6 +104,8 @@ crlify = { path = "../../utils/crlify" }
 icu = { path = "../../components/icu" }
 postcard = "1"
 simple_logger = { version = "4.1.0", default-features = false }
+num-bigint = { version = "0.4.4", default-features = false }
+num-rational = { version = "0.4", default-features = false }
```

Comment:
@Manishearth This diff appeared from trying to merge `release/1.4` into `main`. This is the dev-dependencies section, but I also see these deps in the dependencies section above. Please advise.

Reply:
I suggest editing out these two lines and `commit --amend`.

Reply:
Ok so you confirmed that these lines added on release/1.4 should not be merged to main? If so I will proceed with that understanding (`commit --amend` or a fresh PR)

Reply:
Yes.